helix-core/src/main/java/org/apache/helix/task/TaskDriver.java

```diff
@@ -1173,7 +1176,7 @@ public TaskState pollForWorkflowState(String workflowName, long timeout,
 
     if (ctx == null || !allowedStates.contains(ctx.getWorkflowState())) {
       throw new HelixException(String.format(
-          "Workflow \"%s\" context is empty or not in states: \"%s\", current state: \"%s\"",
+          "Workflow %s context is empty or not in states: %s, current state: %s.",
```

Comment:
The quote gives us clear range in naming. If you dont like to see the escaper, then let's have "()" to bound the names.

Reply:
TFTR. Updated. It was a missed typo have. Thanks for catching this.

Reply:
I removed the quotation mark because I didn't find in anywhere else. I think all logs should have the same format.  Are you suggesting add "()" to bound all ID and names in all the logs?  
I personally find space is enough. Please do suggest on this since I do not have that mush Helix debugging experience.

Reply:
With some quote or bounding chars can help us locate the names. Sometimes the job name or workflow name could be very long.

Reply:
Thanks for the suggestion. If having bounding chars is more preferable, I think it is better to have another PR to change all TF related files. This PR only touches part of them